airbyte-webapp/src/components/connectorBuilder/Builder/Builder.tsx

```diff
@@ -0,0 +1,55 @@
+import { Form } from "formik";
+import { useEffect, useState } from "react";
+
+import { useConnectorBuilderState } from "services/connectorBuilder/ConnectorBuilderStateService";
+import { usePatchFormik } from "views/Connector/ConnectorForm/useBuildForm";
```

Comment:
As we use the formik patch trick in multiple places now, should we pull it into `core/form`?

Reply:
I took a closer look at this and I think we actually don't need it at all in the builder since this form doesn't get submitted. So I'm just going to remove it from the builder instead

Reply:
Ah but we need it in the AddStreamButton component. I'll move it to core/form then